src/pages/public_registration/LogPanel.tsx

```diff
@@ -0,0 +1,82 @@
+import { PublishingTicket, Ticket, TicketStatus, TicketType } from '../../types/publication_types/ticket.types';
+import { Box, Skeleton, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useQuery } from '@tanstack/react-query';
+import { fetchOrganization } from '../../api/cristinApi';
+import { fetchUser } from '../../api/roleApi';
+import { Registration } from '../../types/registration.types';
+import { getFullName } from '../../utils/user-helpers';
+import { DoiRequestMessagesColumn } from '../messages/components/DoiRequestMessagesColumn';
+import { StyledStatusMessageBox } from '../messages/components/PublishingRequestMessagesColumn';
+import { CompletedPublishingRequestStatusBox } from './action_accordions/CompletedPublishingRequestStatusBox';
+
+const ticketStatusesToShow: TicketStatus[] = ['Completed', 'Closed'];
+const ticketTypesToShow: TicketType[] = ['PublishingRequest', 'DoiRequest'];
+
+interface LogPanelProps {
+  tickets: Ticket[];
+  registration: Registration;
+}
+
+export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
+  const { t } = useTranslation();
+  const resourceOwnerAffiliationId = registration.resourceOwner.ownerAffiliation;
+  const resourceOwnerId = registration.resourceOwner.owner;
+
+  const organizationQuery = useQuery({
+    enabled: !!resourceOwnerAffiliationId,
+    queryKey: ['organization', resourceOwnerAffiliationId],
+    queryFn: resourceOwnerAffiliationId ? () => fetchOrganization(resourceOwnerAffiliationId) : undefined,
+    meta: { errorMessage: t('feedback.error.get_institution') },
+    staleTime: Infinity,
+    cacheTime: 1_800_000, // 30 minutes
+  });
+
+  const userQuery = useQuery({
+    enabled: !!resourceOwnerId,
+    queryKey: ['user', resourceOwnerId],
+    queryFn: resourceOwnerId ? () => fetchUser(resourceOwnerId) : undefined,
+    retry: 0,
+    staleTime: Infinity,
+    cacheTime: 1_800_000, // 30 minutes
+  });
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '0.5rem' }}>
+      {registration && (
+        <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
+          <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+            <Typography>{t('common.created')}:</Typography>
+            {organizationQuery.isLoading || userQuery.isLoading ? (
+              <Skeleton sx={{ width: '4rem' }} />
+            ) : (
+              <Typography>
+                {organizationQuery.data ? organizationQuery.data?.acronym : t('common.unknown')}
+                {userQuery.data ? `, ${getFullName(userQuery.data.givenName, userQuery.data.familyName)}` : ''}
```

Comment:
kan diskuteres om dette egentlig skulle vært 2 stk <Typography>, spesielt om det skal være på hver sin linje. Your call

Reply:
nvm, ser nå at det der delt opp i komma her

Reply:
Det ska være på samme linje, separert med komma. Men brytes såklart når teksten blir for lang, noe som sikkert skjer i de fleste tilfeller. Vurderte å force personnavn til å være på samme linje, men da blir det kluss med bredden på loggen.